.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,23 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - name: npm install, audit, lint and test
+      run: |
+        npm ci
+        npm audit
```

Comment:
Do you mind rebasing this branch and re-pushing? I think the removal of `npm audit` in the other PR I just approved and merged is causing a conflict here.

Reply:
Done with 82c169e7efe41a0cdc89d90b5dbdb513156c4c8e